addons/binding/org.openhab.binding.wso2iots/src/main/java/org/openhab/binding/wso2iots/internal/jsonTemp/wso2iotsResponseTemp.java

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
if responses for each sensor hangs under different json branch, you should be able to actually use a single class to deserialize this data, with all branches in it. gson will fill in the values which it will find in the incoming json.